enderio-machines/src/main/java/com/enderio/machines/common/blockentity/FluidTankBlockEntity.java

```diff
@@ -2,9 +2,13 @@
 
 import com.enderio.base.common.blockentity.sync.FluidStackDataSlot;
 import com.enderio.base.common.blockentity.sync.SyncMode;
+import com.enderio.machines.common.MachineTier;
+import com.enderio.machines.common.blockentity.base.MachineBlockEntity;
 import com.enderio.machines.common.blockentity.data.sidecontrol.fluid.FluidTankMaster;
 import com.enderio.machines.common.blockentity.data.sidecontrol.item.ItemHandlerMaster;
+import com.enderio.machines.common.blockentity.data.sidecontrol.item.ItemSlotLayout;
 import com.enderio.machines.common.menu.FluidTankMenu;
+import com.sun.jna.platform.win32.WinUser;
```

Comment:
remove import

Reply:
Where the bloody hell did that come from

Reply:
Exactly my thought